src/services/cypherQueryApiService.ts

```diff
@@ -67,27 +67,44 @@ export const CypherQueryApiService = {
   },
 
   /**
-   * Returns all resources
+   * Returns all resources of type included in the type asset list passed in as parameter.
    *
    * @param authContext authorization token for server calls
-   * @param typeAssets the list of type assets that should be included in the cypher query
+   * @param typeFilters the list of requested resource types
    */
-  async getAllResources(authContext: AuthContextType, typeAssets: Asset[]): Promise<ResourceInput> {
-    // TODO: Should only be loaded only resources with valid type? An asset has a valid type if its type exists in
-    //  the 'typeAssets' array passed in as input parameter.
+  async getAllResources(authContext: AuthContextType, typeFilters: Asset[]): Promise<ResourceInput> {
```

Comment:
The AuthContext is only required for the token. And this is already set in the AuthContextProvider.tsx in line 60/61. Therefore, authContext is no longer required as a parameter

Reply:
done